prow/cluster/monitoring/alertmanager-prow_secret.yaml

```diff
@@ -13,13 +13,15 @@ stringData:
     route:
```

Comment:
We'll need to update and deploy this file manually because of the `{{ api_url }}`, right?

Reply:
Yes